Doc/library/io.rst

```diff
@@ -215,7 +215,7 @@ High-level Module Interface
               return f.read()
 
    In this example, an :class:`EncodingWarning` is emitted for the caller of
-   ``read_text()``.
+   :meth:`!read_text`.
```

Comment:
I think this should stay as ``read_text()``; it refers to the exemplar function defined immediatley above and not to any object description.

 ```suggestion
   ``read_text()``
```

Reply:
I agree. I had an earlier variant where I converted these to the meth markup, but I changed my mind. Probably forgot to revert this one. Thanks for noticing.

Reply:
Did you change your mind on this one? xref [348c115](https://github.com/python/cpython/pull/107903/commits/348c1155d111839a06ad49a5c44a8e6ef435b136)